recipes/pygit2/bld.bat

```diff
@@ -0,0 +1,3 @@
+set LIBGIT2=%LIBRARY%
+"%PYTHON%" setup.py install
```

Comment:
Please add the options `--single-version-externally-managed --record record.txt` after `install`.